assets/src/components/detours/detourMap.tsx

```diff
@@ -70,7 +77,7 @@ interface DetourMapProps {
    * Callback fired when the map is clicked.
    * @param point
```

Comment:
Your description of the considerations you made while renaming this function was very helpful. Thanks for outlining it all. I may be easily swayed, but it does seem sound to me! I'd adjust this comment to suit the new purpose.